tests/data_protection/oadp/test_velero.py

```diff
@@ -87,14 +92,18 @@ def test_restore_multiple_namespaces(
 def test_backup_vm_data_volume_template_with_datamover(
     rhel_vm_with_data_volume_template, velero_restore_first_namespace_with_datamover
 ):
+    wait_for_running_vm(vm=rhel_vm_with_data_volume_template)
```

Comment:
why is this needed? `create_rhel_vm` is calling `running_vm` which calls `wait_for_running_vm`

Reply:
If I recall correctly, the file check in the next line occasionally failed because the SSH server was not active on the machine. The "waiting_for_running_vm" function resolved this issue. However, I can remove it if you prefer, as these changes are unlikely to be accepted until we test them on bare metal in a PSI environment.

Reply:
please remove, and if it fails we need to inderstand why as at this point in the flow, ssh should arelady be running